examples/rdkafka_performance.c

```diff
@@ -3,24 +3,24 @@
  *
  * Copyright (c) 2012, Magnus Edenhill
  * All rights reserved.
- * 
+ *
```

Comment:
perhaps there is less reason to be scared by the line count of this PR than I thought! white space changes ideally shouldn't be here.

Reply:
I generally agree, but this example is so rarely visited that I took the opportunity to fix its style when there was other business in this area. 🤷‍♂